Added security.md (#345)
elastisys/compliantkubernetes-apps: SECURITY.md

# Reporting security issues

If you think you have found a security vulnerability, please send a report to [security@elastisys.com](mailto:security@elastisys.com).

Please encrypt your message to us; please use our PGP key. The key fingerprint is:

31AA7FFF035F50788C01294B6F878E4273E4D4B9

The key is available from [keyserver.ubuntu.com](https://keyserver.ubuntu.com/pks/lookup?search=0x31AA7FFF035F50788C01294B6F878E4273E4D4B9&fingerprint=on&op=index).

> **Important:** We ask you to not disclose the vulnerability before it have been fixed and announced, unless you received a response from the Elastisys security team that you can do so.

Reviewer: Maybe `has`? I'm not sure ^_^
```suggestion
**Important:** We ask you to not disclose the vulnerability before it has been fixed and announced, unless you received a response from the Elastisys security team that you can do so.
```

## Security announcements

> You can subscribe to email updates via an [mail list](http://eepurl.com/htQ4Jv).

Reviewer: Maybe?
```suggestion
You can subscribe to our [mailing list](http://eepurl.com/htQ4Jv) to receive updates via mail.
```
